### PR TITLE
document how to add a new translation

### PR DESCRIPTION
**Pull request by bnchdrff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ import { findById } from '@streetcredlabs/categories';
 
 ## API
 
+The following top-level exports are available:
+
 ### Languages
 
 This TypeScript enum contains all available languages.
```
Comment (tylergaw) [inline README.md]: This was already in place, but is it missing an example or more detail? I'm not sure what it's saying

File: README.md [hunk 2]
```diff
@@ -125,13 +184,13 @@ yarn test
...
+### Run build once:
 
 ```bash
 yarn build
 ```
 
-#### Generate translation files
+### Generate translation files
 
 `yarn translations`
```
Comment (tylergaw) [inline README.md]: I noticed this wasn't used in the step-by-step above. Do we still need it?

Comment (bnchdrff) [reply README.md]: added a clarification to the translations section -- this command should still be done during a release so we should leave it down there

File: README.md (hunk 1 above)
Comment (bnchdrff) [reply README.md]: just added an example

**Comment by bnchdrff** [comment]
@tylergaw 

> This is great. I went through the step-by-step and everything went as described. I hit a couple things that made me pause for a second and could trip other people up. Might be fine, but listing them here in case there's room to clarify in the docs.
> 
>     1. After running `yarn lingui add-locale io` I got a message that said:
> 
> 
> ```
> use "yarn lingui add-locale extract" to extract messages
> ```
> 
> That command is different than our next step of `yarn lingui extract io`. Is that just lingui saying to do something different than our process? I didn't run their suggested command and ran our step two instead; `yarn lingui extract io`

yeah that's a bug in the `lingui` cli! super annoying.

> 
>     1. After running `yarn lingui extract io` I got a message that said:
> 
> 
> ```
> [BABEL] Note: The code generator has deoptimised the styling of "src/data/categories.ts" as it exceeds the max of "500KB".
> ```
> 
> Likely not an issue, but was unsure if I'd done something wrong.
> 

that's due to rollup config.. not sure how to disable that message.

>     1. There's a line break at `yarn lingui compile --typescript` that makes it look like there should be a space after `--`. Maybe that would be more clear as a triple back tick code fence instead?

i'll do that now

**Comment by tylergaw** [comment]
👍 excellent, thanks for the changes. lgtm

**Pull request merged**